enterprise/backend/test/metabase_enterprise/advanced_permissions/api/setting_test.clj

```diff
@@ -152,59 +105,6 @@
               (set-slack-settings :crowberto 200)
               (get-manifest :crowberto 200))))))))
 
-(deftest ldap-api-test
```

Comment:
Removed these tests since they're no longer relevant (setting managers don't have access to auth settings)